src/test/java/org/commcare/formplayer/junit/request/MockRequest.kt

```diff
@@ -37,7 +38,7 @@ open class MockRequest<B, out T : Any>(
             .contentType(MediaType.APPLICATION_JSON)
             .cookie(Cookie(Constants.POSTGRES_DJANGO_SESSION_ID, "derp"))
             .with(SecurityMockMvcRequestPostProcessors.csrf())
-            .with(SecurityMockMvcRequestPostProcessors.user("user"))
+            .with(SecurityMockMvcRequestPostProcessors.user(HqUserDetailsBean("domain", "user")))
```

Comment:
Without this some tests fail [at this line](https://github.com/dimagi/formplayer/blob/master/src/main/java/org/commcare/formplayer/util/RequestUtils.java#L105) as we are casting `principal` to `HqUserDetailsBean`